### ISSUE TITLE
Retain state of files

### ISSUE THREAD

**Issue by shuklaalok7** [lainsce/quilter#266]
This is a suggestion. I really like this feature in other text editors.

The feature is to maintain state of opened files, even the unsaved ones. So, when the program is relaunched even after a reboot, it opens with all the files that were opened previously. This also means implementing autosave.

As an example, Notepad++ does it well.

Thanks!

**Comment by lainsce** [comment]
Quilter does do it. Are you in the newest version available?

**Comment by shuklaalok7** [comment]
I have the latest version available from the AppCenter. I had an unsaved file and when I closed and reopend Quilter, that file was gone.

**Comment by lainsce** [comment]
I'll look into it.

**Comment by janxkoci** [comment]
Does it mean you had autosave off?

**Comment by shuklaalok7** [comment]
@janxkoci Here is a [screencast](https://youtu.be/GalMkVIWK2A) showing all the preferences and also demostrating the issue I am facing.

**Comment by lunfardista** [comment]
Something similar happened to me.
I really like Quilter. I used it in an earlier version and some bug made that a writing that I was working in resulted lost. I installed again recently, and I liked the new version even more. So I was using it for some days and about to make a donation for this useful and appealing app that, again, I really like... But for the second time something happened and I lost my writing. There is not going to be a third try in the near future. The reading time counter is not working (or I don't understand how it works), but in the end it doesn't matters that much, the autosaving bug is the deal breaker.
Keep with the great work, this app is very very nice. I'll check it again in some time. 
Thanks!
